build/charts/theia/values.yaml

```diff
@@ -0,0 +1,101 @@
+      tag: "latest"
+  secret:
+    # -- ClickHouse username. It will be stored in a secret.
+    username: "clickhouse_operator"
+    # -- ClickHouse password. It will be stored in a secret.
+    password: "clickhouse_operator_password"
+  service:
+    # -- The type of service exposes ClickHouse. It can be one of ClusterIP, NodePort or LoadBalancer.
+    type: ClusterIP
+    # -- TCP port number for the ClickHouse service.
+    tcpPort: 9000
+    # -- TCP port number for the ClickHouse service.
+    httpPort: 8123
+  # -- ClickHouse storage size. Can be a plain integer or as a fixed-point
+  # number using one of these quantity suffixes: E, P, T, G, M, K. Or the
+  # power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+  storageSize: "8Gi"
+  # -- Time to live for data in the ClickHouse. Can be a plain integer using
+  # one of these unit suffixes SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, QUARTER,
+  # YEAR.
+  ttl: 1 HOUR
+  persistentVolume:
+    # -- Enable deploying the ClickHouse with Persistent Volumes.
+    enable: false
+    # -- Persistent Volume Provisioner. Required if Persistent Volumes is enable.
+    # It must be one of "StorageClass", "Local", "NFS".
+    provisioner: "Local"
+    # -- The local path. Required when Persistent Volumes is enable and the
+    # provisioner is "Local".
+    localPath: "/data"
+    # -- Affinity for the Local Persistent Volume. Required when Persistent
+    # Volumes is enable and the provisioner is "Local". Please make sure the
+    # Node used to store the ClickHouse data satisfy the affinity requirements.
+    affinity: 
+      required:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: antrea.io/clickhouse-data-node
+            operator: Exists
+    # -- The NFS server hostname or IP address. Required when Persistent Volumes
+    # is enable the provisioner is "NFS".
+    nfsHost: ""
+    # -- The path exported on the NFS server. Required when Persistent Volumes
+    # is enable the provisioner is "NFS".
+    nfsPath: ""
+    # -- The StorageClass used to dynamically provision the Persistent Volume.
+    # Required when Persistent Volumes is enable the provisioner is "StorageClass".
+    storageClass: ""
+grafana:
+  # -- Container image to use for the Grafana.
+  image: 
+    repository: "projects.registry.vmware.com/antrea/theia-grafana"
+    pullPolicy: "IfNotPresent"
+    tag: "8.3.3"
+  secret:
+    # -- Grafana username. It will be stored in a secret.
+    username: "admin"
+    # -- Grafana password. It will be stored in a secret.
+    password: "admin"
+  service:
+    # -- The type of service exposes Grafana. It must be one of NodePort or LoadBalancer.
+    type: NodePort
+    # -- TCP port number for the Grafana service.
+    tcpPort: 3000
+  # -- Grafana plugins to install.
+  installPlugins: "https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.0.zip;theia-grafana-sankey-plugin,grafana-clickhouse-datasource 1.0.1"
```

Comment:
maybe it would be more user-friendly to support a YAML array here. You can then build the string when rendering the template.

Reply:
Using the YAML array now, thanks!